app/src/marketplace/containers/ProjectEditing/DeleteProject.tsx

```diff
@@ -27,7 +28,7 @@ const DeleteProject = () => {
                 kind="destructive"
                 onClick={async () => {
                     try {
-                        await deleteProject()
+                        await deleteProject(project?.id || undefined)
```

Comment:
Since the transaction list toast is now part of the `deleteProject` service utility (more backendish, less reacty), we can drop this piece from the controller and use `deleteProject` directly.

Pretty cool, eh?!